pkg/kubelet/kubelet.go

```diff
@@ -823,6 +826,11 @@ type Kubelet struct {
 
 	// the number of allowed pods per core
 	podsPerCore int
+
+	// enableControllerAttachDetach indicates the Attach/Detach controller
+	// should manage attachment/detachment of volumes scheduled to this node,
+	// and disable kubelet from executing any attach/detach operations
```

Comment:
What about static pods?  I thought we had to maintain backward compat for those.